src/cloud/digitalocean/provisioner.ts

```diff
@@ -127,17 +170,9 @@ class Provisioner {
           });
         }
       }
-      return Promise.reject({
-        'errcode': 'VM_DNE',
-        'message': 'Droplet ' + name + ' doesnt exist'
+      return Promise.resolve({
```

Comment:
I think it's probably better to reject if the droplet doesn't exist; this will help simplify `destroyServer_` and `start`. What do you think?